server/api/dataset.py

```diff
@@ -252,51 +263,36 @@ def submitReviewImages(self, dataset, params):
     @describeRoute(
         Description('Register metadata with a dataset.')
         .param('id', 'The ID of the dataset.', paramType='path')
-        .param('uploadFolderId', 'The ID of the folder that contains metadata '
-               'in a .csv file.')
+        .param('metadataFileId', 'The ID of the .csv metadata file.')
     )
     @access.user
-    @loadmodel(model='dataset', plugin='isic_archive', level=AccessType.READ)
+    @loadmodel(model='dataset', plugin='isic_archive', force=True)
```

Comment:
Why is this? I would think that any user uploading metadata should at least be able to `READ` the dataset.